tests/garage/envs/test_grayscale_env.py

```diff
@@ -16,7 +16,8 @@ def setUp(self):
         self.env = mock.Mock()
```

Comment:
Same question here. Could you use DummyEnv here? If not, I suggest we can write a class for the mock env because it looks like you have used this a lot in different tests.

Reply:
DummyEnv is like
```python
class DummyEnv(gym.Env):
    @property
    def observation_space(self):
        return gym.spaces.Box(
            low=-np.inf, high=np.inf, shape=(1, ), dtype=np.float32)

    @property
    def action_space(self):
        return gym.spaces.Box(
            low=-5.0, high=5.0, shape=(1, ), dtype=np.float32)

    def reset(self):
        return np.zeros(1)

    def step(self, action):
        return Step(
            observation=np.zeros(1), reward=np.random.normal(), done=True)
```

I think DummyEnv is essential same as the mock env we have and it is more convenient. @ryanjulian What do you think?

Reply:
There are DummyEnvs in tests.fixtures.envs. You can add more if need.